frontend/packages/ux-editor/src/converters/containerComponentConverters/internalContainerComponentToExternal.ts

```diff
@@ -12,6 +13,6 @@ export const internalContainerComponentToExternal = (
   return {
     ...propertiesToKeep,
     children,
-    type: internalContainerComponent.type,
-  };
+    type: internalContainerComponent.type as ContainerComponentType,
+  } as any;
```

Comment:
@TomasEng I was not able to figure out how to solve this casting in a good way. Any suggestions?

Reply:
This is an interesting case. I couldn't find any solution without casting. Is there any reason for replacing the type of `internalContainerComponent` with `FormItem`? I tried setting it back to `FormContainer`, and then the typecheck was happy with casting the result to `ExternalContainerComponent`.
```
export const internalContainerComponentToExternal = (
  internalContainerComponent: FormContainer,
  children: string[],
): ExternalContainerComponent => {
  const propertiesToKeep = { ...internalContainerComponent };
  delete propertiesToKeep.itemType;
  delete propertiesToKeep.propertyPath;
  delete propertiesToKeep.pageIndex;
  return {
    ...propertiesToKeep,
    children,
  } as ExternalContainerComponent;
};
```